### ISSUE TITLE
Just a warning that the package doesn't work with Transformers 4.25.1

### ISSUE THREAD

**Issue by JulesGM** [allenai/RL4LMs#22]
Looks like generation_beam_constraints doesn't exist or has been moved?

<img width="1247" alt="Screen Shot 2022-12-16 at 4 25 29 PM" src="https://user-images.githubusercontent.com/3231217/208191636-a20e7224-3c1d-42e9-931f-3494ee3da99c.png">

**Comment by rajcscw** [comment]
yeah true, if we are to upgrade transformers, we need to update hf_generation_utils.py too.
